util/include/DigestImpl.ipp

```diff
@@ -55,7 +71,7 @@ class DigestHolder {
   DigestHolder() { std::memset(d, 0, DIGEST_SIZE); }
   DigestHolder(unsigned char initVal) { std::memset(d, initVal, DIGEST_SIZE); }
   DigestHolder(const char* other) { std::memcpy(d, other, DIGEST_SIZE); }
-  DigestHolder(char* buf, size_t len) { DigestUtil::compute(buf, len, (char*)d, DIGEST_SIZE); }
+  DigestHolder(char* buf, size_t len) { CREATOR::compute(buf, len, (char*)d, DIGEST_SIZE); }
```

Comment:
Implement it as:
CREATOR digest_creator;
digest_creator.compute(buf, len, (char*)d, DIGEST_SIZE);

Reply:
Addressed https://github.com/arc-vmware/concord-bft/pull/3/commits/93e565e22eae96ba5c40756c974a391f8bcb282d